stdlib/public/core/StringStorage.swift

```diff
@@ -507,7 +510,7 @@ extension _StringStorage {
   @_effects(releasenone)
   private func _postRRCAdjust(newCount: Int, newIsASCII: Bool) {
     let countAndFlags = CountAndFlags(
-      count: newCount, isASCII: newIsASCII)
+      mortalCount: newCount, isASCII: newIsASCII)
```

Comment:
"mortalCount" is the most metal variable name I've seen in a while

Reply:
🤘